src/linearmixedmodel.jl

```diff
@@ -895,7 +893,7 @@ end
 
 The penalized, weighted residual sum-of-squares.
 """
-pwrss(m::LinearMixedModel{T}) where {T} = abs2(last(last(m.L)))::T
+pwrss(m::LinearMixedModel{T}) where {T} = abs2(last(last(m.L))::T)
```

Comment:
moving the type assertion inward means that `abs2` now dispatches on an inferred type